worker/firewaller/firewaller.go

```diff
@@ -1028,6 +1064,52 @@ func (fw *Firewaller) flushGlobalPorts(rawOpen, rawClose firewall.IngressRules)
 	return nil
 }
 
+func (fw *Firewaller) flushModel() error {
+	if fw.environModelFirewaller == nil {
```

Comment:
We should not do this here - we should check for this before even starting the watcher.

Reply:
Good idea. No point watching if we can't do anything with that info